test/types/base.test.ts

```diff
@@ -69,3 +69,5 @@ function setAsObject() {
 
   expectError(mongoose.set({ invalid: true }));
 }
+
+const x: { name: string } = mongoose.omitUndefined({ name: 'foo' });
```

Comment:
consider also adding a test that actually removes something

Reply:
A reasonable suggestion, but not really worth it for TypeScript since we don't actually run the TypeScript tests, we're just testing the type

Reply:
right, forgot the typescript type didnt actually modify the input type